lambda_functions/automate_s2_nbar_rolling_archive/execute_s2nbar/test_handler.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+from unittest.mock import call
+from mock import patch
+from handler import automate_rolling_archive, handler
+
+
+class TestLambdaFunction(unittest.TestCase):
+
+    @patch('handler.boto3.client')
+    @patch('handler.automate_rolling_archive')
+    def test_handler(self, mock_client, mock_automate_rolling_archive):
+        event = {
+            "Records": [
+                {
+                    "Sns": {
+                        "Message": "Level Sentinel2 Definitive process completed\n",
+                        "Timestamp": "Timestamp",
+                        "MessageId": "Message_1234"
+                    },
+                },
+            ]
+        }
+
+        # Run the function under test
+        handler(event, None)
+
+        mock_automate_rolling_archive.assert_called
```

Comment:
This isn't a valuable test, it just checks to see if an internal function is called and that the lambda doesn't crash...

Reply:
Reworked on the test script to aid functionality testing